### PR TITLE
Feature sb 37 header and footer

### PR DESCRIPTION
**Pull request by yuliiaaaaa**
1. Created 2 types of header:for unregistered users and logged
2. Added fonts, colors,icons,translation
3. Created 2 types of footer:for unregistered users and logged
4. Added some changes in imports,file names

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="/src/index.tsx"></script>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Vite + React + TS</title>
```
Comment (lyudad) [inline index.html]: your team name

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { colors } from "~/libs/constants/color.ts";
+import { footerColumnsLogIn } from "~/libs/constants/footer-links.ts";
+import {
+	socialMediaDataLogIn,
+	socialMediaDataLogOut,
+} from "~/libs/constants/social-media-icons.ts";
+
+export const Footer: React.FC = () => {
+	const { t } = useTranslation();
+	const isLoggIned = false;
```
Comment (lyudad) [inline src/components/Footer/Footer.tsx]: what is it?) no comments if its temporary, we cant left as it is, no one will remember after few days this strange logic

File: src/components/Header/main-logo.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { useTranslation } from "react-i18next";
+
+import { CircularElement } from "~/components/footer/circular.tsx";
+import { LogoContainer, StyledTypography } from "~/components/footer/styles.ts";
+import { Link } from "~/components/header/link.tsx";
+import { colors } from "~/libs/constants/color.ts";
+
+export const MainLogo: React.FC = () => {
+	const { t } = useTranslation();
+	return (
```
Comment (lyudad) [inline src/components/Header/main-logo.tsx:12]: empty line above, we need some separation, to easier read

File: src/libs/constants/social-media-icons.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		iconColor: colors.WHITE,
+	},
+	{
+		IconComponent: MailOutlineSharpIcon,
+		backgroundColor: colors.BLACK,
+		href: "#",
+		iconColor: colors.WHITE,
+	},
+];
+const socialMediaDataLogOut = [
```
Comment (lyudad) [inline src/libs/constants/social-media-icons.ts:29]: empty line above

File: src/components/Header/main-logo.tsx (hunk 3 above)
Comment (Vfivevv) [reply src/components/Header/main-logo.tsx:11]: Seems like linter/prettier deleted the space here. Check if that’s the case, and if so, let me know - i will fix the linter.
@yuliiaaaaa

Comment (VladManukian) [reply src/components/Header/main-logo.tsx:12]: Seems like linter/prettier deleted the space here. Check if that’s the case, and if so, let me know - i will fix the linter. @yuliiaaaaa

File: index.html (hunk 1 above)
Comment (yuliiaaaaa) [reply index.html]: done

File: src/components/Footer/Footer.tsx (hunk 2 above)
Comment (yuliiaaaaa) [reply src/components/Footer/Footer.tsx]: done

File: src/libs/constants/social-media-icons.ts (hunk 4 above)
Comment (yuliiaaaaa) [reply src/libs/constants/social-media-icons.ts:29]: Done

File: src/components/Header/main-logo.tsx (hunk 3 above)
Comment (yuliiaaaaa) [reply src/components/Header/main-logo.tsx:12]: done

File: src/components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+} from "~/libs/constants/social-media-icons.ts";
+import { useAppSelector } from "~/redux/hooks/use-app-selector.ts";
+import { RootState } from "~/redux/store.ts";
+
+export const Footer: React.FC = () => {
+	const { t } = useTranslation();
+	const isLoggIned = Boolean(
+		useAppSelector((state: RootState) => state.auth.accessToken),
+	);
+	const footerBgColor = isLoggIned ? colors.GREYRISH_RED : colors.CREME;
```
Comment (lyudad) [inline src/components/Footer/Footer.tsx:32]: the empty line above. please do not make one text from code, its difficult to read even book has paragraphs

**Pull request merged**